### PR TITLE
Support additional route group attributes

### PR DESCRIPTION
**Pull request by rwinzhang**
Sometimes there are multiples authentication guard defined, it is better if we configure it

### CODE REVIEW

**Comment by rwinzhang** [comment]
@rebing it is actually introduced in Laravel v5.2 –– [changelog](https://laravel.com/docs/5.2/releases#laravel-5.2).
If you want to support v5.1, we can use more general config likes, 
```
'extra_route_groups' => ['guard' => 'api']
```

**Comment by rebing** [comment]
Yes, I think the `extra_route_groups` is a better solution for maintaining backwards compatibility.

**Comment by rwinzhang** [comment]
@rebing I have update my commit

File: src/config/config.php [hunk 1]
```diff
@@ -46,6 +46,9 @@
     // Any middleware for the graphql route group
     'middleware' => [],
 
+    // Additional route group attributes
+    'route_group_attributes' => ['guard' => 'api'],
```
Comment (rebing) [inline src/config/config.php]: This should not be added as a default option. Add it to the comment as an example instead

Comment (rwinzhang) [reply src/config/config.php]: done

**Pull request merged**